flang/lib/Lower/IO.cpp

```diff
@@ -211,6 +211,8 @@ static mlir::FuncOp getOutputFunc(mlir::Location loc,
     return getIORuntimeFunc<mkIOKey(OutputDescriptor)>(loc, builder);
   if (Fortran::lower::CharacterExprHelper::isCharacter(type))
     return getIORuntimeFunc<mkIOKey(OutputAscii)>(loc, builder);
+  if (type.isa<fir::SequenceType>())
+    return getIORuntimeFunc<mkIOKey(OutputDescriptor)>(loc, builder);
   // TODO: handle arrays
```

Comment:
That's a lot of new work!

These two TODOs look obsolete.

Reply:
Arg. I'll fix up the comments.